packages/loopring_v3/circuit/Utils/Data.h

```diff
@@ -14,14 +16,130 @@ using json = nlohmann::json;
 namespace Loopring
 {
 
-enum class BlockType
-{
-    RingSettlement = 0,
+static auto dummySpotTrade = R"({
+    "fFillS_A": 0,
+    "fFillS_B": 0,
+    "orderA": {
+        "accountID": 0,
+        "allOrNone": false,
+        "amountB": "79228162514264337593543950335",
+        "amountS": "79228162514264337593543950335",
+        "buy": true,
+        "feeBips": 0,
+        "maxFeeBips": 0,
+        "orderID": "0",
+        "rebateBips": 0,
+        "tokenS": 0,
+        "tokenB": 1,
+        "validSince": 0,
+        "validUntil": 4294967295
+    },
+    "orderB": {
+        "accountID": 0,
+        "allOrNone": false,
+        "amountB": "79228162514264337593543950335",
+        "amountS": "79228162514264337593543950335",
+        "buy": true,
+        "feeBips": 0,
+        "maxFeeBips": 0,
+        "orderID": "0",
+        "rebateBips": 0,
+        "reduceOnly": 0,
+        "tokenS": 1,
+        "tokenB": 0,
+        "validSince": 0,
+        "validUntil": 4294967295
+    }
+})"_json;
+
+static auto dummyTransfer = R"({
+    "fromAccountID": 0,
+    "toAccountID": 2,
```

Comment:
It seems acountID=2 is special. Do we have to reserve it?

Reply:
No, it's just the first valid accountID that can be used in a tx. The circuit disallows transactions from the accounts used by the protocol pool (and still in this version, the index). This is to disallow e.g. a tx that sets the owner of the protocol pool to some address. Without the index, it can be any accountID > 0.

Reply:
I thought only account 0 and 1 are special, are you saying account 2 is also special and cannot be used by a normal user? Or we can even use any account id other than 0 and 1 here?

No matter  we use 2 or 3 here, I think the relayer better not to assign this account ID to a user.

Reply:
It is a normal account which is why it is used.

> Or we can even use any account id other than 0 and 1 here?

Yes.

> No matter we use 2 or 3 here, I think the relayer better not to assign this account ID to a user.

It shouldn't be a problem. Especially in this case because these inputs are not used and have no effect on whatever account is used, they are just there to make the circuits valid. 

There are some places where this account is used, not because of this code specifically but because it's the default account. NOOPs for example which don't change anything for any account will "use" the account, but just to make the circuit valid (leaving the account unchanged). So it could make sense not to use it for those reasons, but I don't think it should matter.